### ISSUE TITLE
formula: provide bottle

### ISSUE THREAD

**Issue by efiop** [iterative/homebrew-dvc#16]
https://docs.brew.sh/Bottles

**Comment by AlJohri** [comment]
Once you submit the formula to homebrew-core (https://github.com/iterative/homebrew-dvc/issues/13) (which I believe will automatically provide a bottle?) is there any reason to keep both the brew formula and a cask around? It seems like the additional choice may just cause confusion.

I personally use the `brew cask install dvc` right now because its much faster than `brew install dvc`- however a bottle might change that.

`brew cask install dvc` has the unfortunate downside of being a package installer so it requires inputting a password every time I upgrade. a minor inconvenience, but given the fast update cycle, its definitely a mild deterrent to upgrading frequently

**tl;dr** proposal: remove cask once homebrew-core formula is accepted and bottle is made and direct everyone to simply `brew install dvc`. leave around the homebrew-dvc tap which will print a message redirecting users to run something along the lines of: `brew untap iterative/dvc && brew install dvc`

**Comment by efiop** [comment]
@AlJohri Great point! I don't quite remember the details, but we've had some issues with formulas a few years ago and so cask was the first one that we've created. Another nice part about cask is that we ship an all-in-one package, with all dependencies included, so theoretically it should work in more cases than formula that relies on some environment. But I do agree that at this point there is not much sense in carrying a cask, so we might remove it in the future. That being said, supporting cask doesn't cost us anything and doesn't create any headaches, as it is based on pkg installer that we will probably keep around anyway, so we might as well just leave cask for now as well.

Regarding the bottles in general, indeed looks like once dvc will be accepted into homebrew-core, their bot will build the wheels for us. If we don't get accepted, we could build bottles ourselves too in this repo. I'll try to submit to homebrew-core ASAP. Thanks for the feedback! :slightly_smiling_face:

**Comment by efiop** [comment]
@AlJohri dvc is now part of the default formulae, so you can install it with `brew install dvc`, which will automatically use a bottle to install it.

**Comment by efiop** [comment]
And this `homebrew-dvc` repo will stay for dev purposes and it won't provide bottles.
